src/basic/time-util.h

```diff
@@ -77,6 +77,7 @@ typedef enum TimestampStyle {
 usec_t now(clockid_t clock);
 nsec_t now_nsec(clockid_t clock);
 
+usec_t map_clock_usec_internal(usec_t from, usec_t from_base, usec_t to_base);
 usec_t map_clock_usec(usec_t from, clockid_t from_clock, clockid_t to_clock);
```

Comment:
It's strange to make function called "…_internal" public. It's also not clear how the two functions differ.
I think ` map_clock_usec_internal` should be renamed to `map_clock_usec_raw` or something like that.